tasks/utils/contract-functions.ts

```diff
@@ -1,5 +1,5 @@
 import '@nomiclabs/hardhat-ethers';
-import '@/plugins/fireblocks';
+import '@fireblocks/hardhat-fireblocks';
```

Comment:
do you know if these tasks work with the fireblocks signer plugin?

Reply:
they sure doooo